plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -794,122 +800,145 @@ def get_want_authentication_policy_server(self, auth_policy_server):
+            if not authentication_port:
+                authentication_port = 1812
 
-        auth_server.update({"authenticationPort": authentication_port})
+            if not str(authentication_port).isdigit():
+                self.msg = "The 'authentication_port' should contain only digits."
+                self.status = "failed"
+                return self
 
-        accounting_port = auth_policy_server.get("accounting_port")
-        if not accounting_port:
-            accounting_port = 1813
+            if not 1 <= authentication_port <= 65535:
+                self.msg = "The 'authentication_port' should be from 1 to 65535."
+                self.status = "failed"
+                return self
 
-        if not str(accounting_port).isdigit():
-            self.msg = "The 'accounting_port' should contain only digits."
-            self.status = "failed"
-            return self
+            auth_server.update({"authenticationPort": authentication_port})
+        else:
+            auth_server.update({"authenticationPort": auth_server_details.get("authenticationPort")})
 
-        if not 1 <= accounting_port <= 65535:
-            self.msg = "The 'accounting_port' should be from 1 to 65535."
-            self.status = "failed"
-            return self
+        if not auth_server_exists:
+            accounting_port = auth_policy_server.get("accounting_port")
+            if not accounting_port:
+                accounting_port = 1813
 
-        auth_server.update({"accountingPort": accounting_port})
+            if not str(accounting_port).isdigit():
+                self.msg = "The 'accounting_port' should contain only digits."
+                self.status = "failed"
+                return self
 
-        retries = auth_policy_server.get("retries")
-        if not retries:
-            retries = "3"
+            if not 1 <= accounting_port <= 65535:
+                self.msg = "The 'accounting_port' should be from 1 to 65535."
+                self.status = "failed"
+                return self
 
-        retries = str(retries)
-        if not retries.isdigit():
-            self.msg = "The 'retries' should contain only from 0-9."
-            self.status = "failed"
-            return self
+            auth_server.update({"accountingPort": accounting_port})
+        else:
+            auth_server.update({"accountingPort": auth_server_details.get("accountingPort")})
 
-        if not 1 <= int(retries) <= 3:
-            self.msg = "The 'retries' should be from 1 to 3."
-            self.status = "failed"
-            return self
+        retries = auth_policy_server.get("retries")
+        if not retries:
+            if not auth_server_exists:
+                auth_server.update({"retries": "3"})
+            else:
+                auth_server.update({"retries": auth_server_details.get("retries")})
+        else:
+            try:
+                retries = str(retries)
```

Comment:
not wure, whey do we need to convert to string first and then in next line back to int.. Can we do the range check first and do it?
```
    if retries < 1 or retries > 3:
        self.msg = "The 'retries' value must be within the range of 1 to 3."
        self.status = "failed"
        return self
```